client/src/components/continuousLegend/index.tsx

```diff
@@ -119,51 +120,98 @@ const continuous = (selectorId: any, colorScale: any, colorAccessor: any) => {
   genesets: (state as any).genesets.genesets,
 }))
 class ContinuousLegend extends React.Component {
+  static watchAsync(props: any, prevProps: any) {
+    return !shallowEqual(props.watchProps, prevProps.watchProps);
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
```

Comment:
Can we type this properly please? In effort to slowly convert Explorer repo to be more typed, we want new code and touched old code to be typed as much as possible 🙏 Happy to help where needed!

context: https://czi-sci.slack.com/archives/C03TR1S2CCU/p1660566509067059